tests/test_inspector.py

```diff
@@ -86,6 +88,44 @@ def add_simple_table(nwbfile: NWBFile):
 
 
 class TestInspector(TestCase):
+    """A common helper class for testing the NWBInspector."""
```

Comment:
Peeled out this class so the helpers for asserting log file contents can be used across the suite

Might be worth moving to `nwbinspector.testing` in a follow-up